itscalledsoccer/client.py

```diff
@@ -339,9 +339,9 @@ def _get_stats(
             self._check_leagues_salaries(leagues)
             if (
                 entity == "teams"
-                and kwargs.get("split_by_teams", None)
-                and kwargs.get("split_by_seasons", None)
-                and kwargs.get("split_by_positions", None)
+                and not kwargs.get("split_by_teams", None)
```

Comment:
Should we check for None instead of any falsy result, or is having all of these options be False also an invalid query?

Reply:
If any of these are not set, then it's an invalid query. I can always change the `None` to `False` too if we want

Reply:
yeah I think we should do that. We may also want to check for the string version of `false` explicitly as well -- not sure if that gets processed as truthy (since it's a valid string) or falsy.